### PR TITLE
[MOBILE-1645] Add HMS Module

### PR DESCRIPTION
**Pull request by Ulrico972**
### What do these changes do?
Adds HMS Module

### Why are these changes necessary?
To keep the react native plugin up to date with the last native features

### How did you verify these changes?
Ryan did the tests because I don't have any Huawei to test

### Anything else a reviewer should know?
Ryan did the tests because I don't have any Huawei to test.
Also the HMS module needs additional steps to work in an app. Without this the app crashes, so Ryan choose to keep it out of the example.

### CODE REVIEW

File: urbanairship-hms-react-native/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Airship HMS React Native
+
+Installs the HMS push provider for Airship SDK.
+
+### Resources
+
+* [Getting started guide](http://docs.airship.com/platform/react-native/)
+* [API docs](http://docs.airship.com/reference/libraries/react-native/latest/index.html)
+
```
Comment (BrianBatchelder) [inline urbanairship-hms-react-native/README.md:9]:
```suggestion
```

File: urbanairship-hms-react-native/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+```
+npm install urbanairship-hms-react-native --save
+```
+
+### Add Huawei dependencies
+
+This plugin does not include the HMS dependency directly, instead you need to follow [HMS setup guide](https://developer.huawei.com/consumer/en/doc/development/HMS-Guides/Preparations#h1-1575707420071) to add it the to app directly.
+
+
+
```
Comment (BrianBatchelder) [inline urbanairship-hms-react-native/README.md:28]:
```suggestion
```

File: urbanairship-react-native/android/build.gradle [hunk 3]
```diff
@@ -6,7 +6,7 @@ def safeExtGet (prop, fallback) {
 }
 
 dependencies {
-    def airshipVersion = "13.1.2"
+    def airshipVersion = "13.2.0"
```
Comment (BrianBatchelder) [inline urbanairship-react-native/android/build.gradle:9]: Does this still need to be here now that it is in `common-build.gradle`?

Comment (Ulrico972) [reply urbanairship-react-native/android/build.gradle:9]: Indeed, this is not needed anymore, thanks 😉

**Pull request merged**